doc/ics/yaml/heinz_entsorgung_de.yaml

```diff
@@ -0,0 +1,15 @@
+title: Heinz-Entsorgung (Landkreis Freising)
+url: https://www.heinz-entsorgung.de/
+howto: |
+  - Goto <https://www.heinz-entsorgung.de/leistungen/haushalte/entsorgungskalender/entsorgungskalender-freising/> and select your location.
+  - Click on `ICAL-Datei`
+  - Select the types of waste you are interested in ("Fraktionen")
+  - Click on `Ok`
+  - Download the ics file
+  - Get the download link address from your browser's download history
+  - Replace the `url` in the example configuration with this link.
```

Comment:
Would be nice to add a line that advices to replace the year with `{%Y}` to allow this configuration to work multiple years. Please do the same in the test cases

Reply:
Thanks for the feedback, you are right. I added a line to the instructions and modified the examples accordingly.